docs/mobile-apps/live-testing/live-mobile-app-testing.md

```diff
@@ -20,7 +20,11 @@ You can upload your app via the Sauce Labs UI or via the REST API. For informati
 To upload an app via the Sauce Labs UI:
 
 1. In Sauce Labs, in the left panel, click **LIVE** and then click **Mobile App**.
-2. Click **App Upload**. You can either drag and drop an application, or browse for and select the file. We currently support APK and IPA files up to 4 GB. Non-app file uploads are not supported in the UI at this time, but can be uploaded through the API.
+2. Click **App Upload**. You can either drag and drop an application, or browse for and select the file. We currently support \*.apk or \*.aab Android app files and \*.ipa or \*.zip iOS app files (\*.zip files are parsed to determine whether a valid \*.app bundle exists). Non-app file uploads are not supported in the UI at this time, but can be uploaded through the API.
+
+:::caution Limited Support for *.aab Files
+At this time, \*.aab files are only supported for Android real device testing.
+:::
```

Comment:
```suggestion
2. Click **App Upload**. You can either drag and drop an application, or browse for and select the file. We currently support \*.apk Android app files and \*.ipa or \*.zip iOS app files (\*.zip files are parsed to determine whether a valid \*.app bundle exists). Non-app file uploads are not supported in the UI at this time, but can be uploaded through the API.
```
Is this what you had in mind to remove for now, @linafarid?

Reply:
What about we add something along the lines of
"support for .aab file is currently not supported via the UI but coming soon" ?